dietpi/dietpi-software

```diff
@@ -4199,6 +4205,111 @@ _EOF_
 			G_EXEC sysctl -w net.core.rmem_max=2500000
 		fi
 
+		if To_Install 206 # microblog.pub
+		then
+			## 1. Create a python environment py3.10 via pyenc
```

Comment:
Does it have so many versioned dependencies that a `pyenv` makes sense? This always means a lengthy compiling step.

Reply:
It has quite a few poetry dependencies. Is there an easier way without having to compile?

Reply:
Ah, I forgot about the minimum Python 3.10 requirement, so no way around then on Debian Buster and Bullseye. I guess on Debian Bookworm it could be skipped, but we can add this at a later time when the pyenv based implementation has been polished.

Suggestion about the little typo stays valid 😉.